miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/service/integration/jira/JiraIssueManager.java

```diff
@@ -58,24 +67,39 @@
  */
 @Component
 public class JiraIssueManager implements IssueTrackerManager {
+
+  private static final DateFormat ISO8601_FORMAT = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSZ");
+  private static final List<Character> SPECIAL_CHARS = Lists.newArrayList('+', '-', '&', '|', '!', '(', ')', '{', '}', '[', ']', '^', '~',
+      '*', '?', '\\', ':');
+
   private String oAuthConsumerKey;
   private String oAuthConsumerSecret;
   private String oAuthSignatureMethod;
 
   private String httpBasicAuthUsername;
   private String httpBasicAuthPassword;
 
-  public String baseTrackerUrl;
+  private String baseTrackerUrl;
   private final String restApiUrl = "/rest/api/";
-  public String jiraRestApiVersion = "2";
-
-  public final String jiraIssueSuffix = restApiUrl + jiraRestApiVersion + "/issue/";
+  private final String jiraRestApiVersion = "2";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this final field static too. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1170)